amundsen_application/static/js/components/SearchPage/SearchFilter/styles.scss

```diff
@@ -15,6 +15,10 @@
       button {
         margin: auto 4px;
       }
+
+      label {
+        margin-bottom: 0;
```

Comment:
Fixes this misalignment issue:

![image](https://user-images.githubusercontent.com/1790900/84552322-37a02b80-acc5-11ea-9d66-f29ba4797963.png)